### ISSUE TITLE
Nova 4: Card with width 2/3 doesn't render correctly

### ISSUE THREAD

**Issue by dwightwatson** [laravel/nova-issues#3828]
- Laravel Version: 9.5.1
- Nova Version: 4.0.0
- PHP Version: 8.1.4

### Description:
I use custom card widths typically in a 1/3 and 2/3 arrangement. After upgrading to Nova 4 it looks like it doesn't render the 2/3 width properly.

<img width="695" alt="image" src="https://user-images.githubusercontent.com/1100408/161889714-36a64243-f69f-4c05-b7ff-660e5284640f.png">

It looks like the 1/3 card gets these classes: `bg-white dark:bg-gray-800 rounded-lg shadow relative px-6 py-4 min-h-40 md:col-span-4 h-full` while the 2/3 card gets these classes: `bg-white dark:bg-gray-800 rounded-lg shadow relative px-6 py-4 h-full` - not that there is no `col-span-*` class.

### Detailed steps to reproduce the issue on a fresh Nova installation:

Create two metrics and add them to the Main dashboard with different widths.

```php
(new \App\Nova\Metrics\NewUsers)->width('1/3'),
(new \App\Nova\Metrics\UsersPerMonth)->width('2/3'),
```

**Comment by RhysLees** [comment]
It almost seems like it's intended to be be removed as there are consts for 1/3 1/2 full in the card

**Comment by dwightwatson** [comment]
Still see it referenced in the docs, and wasn't mentioned in the upgrade guide so makes me think this wasn't intentional - but confirmation would be nice.

**Comment by RhysLees** [comment]
> Still see it referenced in the docs, and wasn't mentioned in the upgrade guide so makes me think this wasn't intentional - but confirmation would be nice. 

Exactly it was best for the trend metrics, sometimes half is too small and full is too large 🤣

**Comment by aaronwjm360** [comment]
It seems that the same issue occurs with the 1/4 width as well.

**Comment by Bradez** [comment]
Looks like this may have been fixed with [v4.0.4](https://nova.laravel.com/releases/4.0.4).
